--- a/cmake/tools/SetupLLVM.cmake
+++ b/cmake/tools/SetupLLVM.cmake
@@ -120,25 +120,25 @@
   find_llvm_command_no_version(CMAKE_AR llvm-lib)
   find_llvm_command_no_version(CMAKE_STRIP llvm-strip)
 else()
-  find_llvm_command(CMAKE_C_COMPILER clang)
-  find_llvm_command(CMAKE_CXX_COMPILER clang++)
-  find_llvm_command(CMAKE_LINKER llvm-link)
-  find_llvm_command(CMAKE_AR llvm-ar)
+  find_llvm_command_no_version(CMAKE_C_COMPILER clang)
+  find_llvm_command_no_version(CMAKE_CXX_COMPILER clang++)
+  find_llvm_command_no_version(CMAKE_LINKER llvm-link)
+  find_llvm_command_no_version(CMAKE_AR llvm-ar)
   if (LINUX)
     # On Linux, strip ends up being more useful for us.
     find_command(
       VARIABLE
         CMAKE_STRIP
       COMMAND
         strip
       REQUIRED
         ON
     )
   else()
-    find_llvm_command(CMAKE_STRIP llvm-strip)
+    find_llvm_command_no_version(CMAKE_STRIP llvm-strip)
   endif()
-  find_llvm_command(CMAKE_RANLIB llvm-ranlib)
+  find_llvm_command_no_version(CMAKE_RANLIB llvm-ranlib)
   if(LINUX)
-    find_llvm_command(LLD_PROGRAM ld.lld)
+    find_llvm_command_no_version(LLD_PROGRAM ld.lld)
     # Ensure vendor dependencies use lld instead of ld
     list(APPEND CMAKE_ARGS -DCMAKE_EXE_LINKER_FLAGS=--ld-path=${LLD_PROGRAM})
     list(APPEND CMAKE_ARGS -DCMAKE_SHARED_LINKER_FLAGS=--ld-path=${LLD_PROGRAM})
   endif()
   if(APPLE)
-    find_llvm_command(CMAKE_DSYMUTIL dsymutil)
+    find_llvm_command_no_version(CMAKE_DSYMUTIL dsymutil)
   endif()
 endif()