--- a/src/host/server.h
+++ b/src/host/server.h
@@ -29,6 +29,7 @@
     void CleanForwardMap(uint32_t sessionId);
     int CreateConnect(const string &connectKey, bool isCheck);
     bool Initial(const char *listenString);
+    void ClearInstanceResource() override;
     void AttachChannel(HSession hSession, const uint32_t channelId) override;
     void DeatchChannel(HSession hSession, const uint32_t channelId) override;
     virtual void EchoToClientsForSession(uint32_t targetSessionId, const string &echo);
@@ -46,7 +47,6 @@
     void *clsServerForClient;
 
 private:
-    void ClearInstanceResource() override;
     void BuildDaemonVisableLine(HDaemonInfo hdi, bool fullDisplay, string &out);
     void BuildForwardVisableLine(bool fullOrSimble, HForwardInfo hfi, string &echo);
     void ClearMapDaemonInfo();
